kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaServiceConfiguration.java

```diff
@@ -182,6 +182,12 @@
     )
     private long offsetsRetentionCheckIntervalMs = OffsetConfig.DefaultOffsetsRetentionCheckIntervalMs;
 
+    @FieldContext(
+            category = CATEGORY_KOP,
+            doc = "send queue size of system client to produce system topic."
+    )
+    private int maxPendingMessages = 10000;
```

Comment:
what about adding "kafka" prefix ?

Reply:
+1 to this. Just left a minor suggestion, you can leave it out. Can we use for example `kafkaMetaMaxPendingMessages`? Otherwise I can easily think of `io.streamnative.pulsar.handlers.kop.KafkaServiceConfiguration#maxReadEntriesNum`, which is a configuration related to client-side.

Reply:
I suggest adding a config whose value is a config file like

```properties
private String kopMetaProducerConfigFile = null;
```

If it's not null, the producer of group coordinator will use the config file to initialize some configurations. This way could be more flexible.

Reply:
@eolivelli @wenbingshen done, changed to kafkaMetaMaxPendingMessages
@BewareMyPower We could add the client config file in another pr. This pr aim to change the default value for send queue size.